.github/workflows/deploy-analytics.yml

```
@@ -0,0 +1,179 @@
+"""GitHub Actions CI/CD pipeline for analytics platform."""
```

actionlint (1.7.7)

workflow is scalar node but mapping node is expected

(syntax-check)